### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 106971: character maps to <undefined>

### ISSUE THREAD

**Issue by MellowEdge** [BEF-BEF/Jerma#1]
Ran with
```
index_content()
search_indexed("garlic")
```
in main.py, got an error midway through:

```
Indexed segment with timestamp 25816.62s -> 25817.50s from file: outDirectories\outDirectories1\Jerma Streams - Dead Rising\transcription.txt
Traceback (most recent call last):
  File "D:\Programs\Jerma-main\main.py", line 45, in <module>
    main()
  File "D:\Programs\Jerma-main\main.py", line 39, in main
    index_content()
  File "D:\Programs\Jerma-main\cleanIndex.py", line 114, in index_content
    index_file(full_path, writer, searcher)
  File "D:\Programs\Jerma-main\cleanIndex.py", line 93, in index_file
    content = f.read()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\encodings\cp1251.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 106971: character maps to <undefined>
```

**Comment by BEF-BEF** [comment]
Hey I just saw this. Did you figure it out

**Comment by MellowEdge** [comment]
Sorry, I forgot to answer. I don't really have any idea how to approach this so I didn't figure it out

**Comment by BEF-BEF** [comment]
I made a new commit today and everything was working when I tested it. You're welcome to try again. I tried to make it easier for people to get started with it. Let me know if you're still having problems. Best way to reach me is to dm .befbef on discord

**Comment by BEF-BEF** [comment]
I found the issue! On Linux f.read defaults to utf-8, but that's not true on Windows. I changed f.read to force it to read utf-8 and it's working on my windows now. There were several other compatibility issues with windows. I fixed them and I'm testing it now, I'll commit the changes soon

**Comment by BEF-BEF** [comment]
I got it working smoothly on Windows now. There were a lot of tweaks that I had to make to reading or writing files because Windows handles things a bit differently than Ubuntu. I tested it on Windows and Ubuntu. It should work fine now. I learned a better way to do things that keeps both Windows and Ubuntu happy. Try following the new quick start guide, it's meant to be pretty straightforward. Reach out on discord if there are any problems.
